docs/recipes/package-management/add-npm-package-to-client.md

```diff
@@ -1,6 +1,6 @@
 # How do I add an NPM package to the Client?
 
-When you want to [call a Javascript library](https://fable.io/docs/communicate/js-from-fable.html) from your Client, it is easy to import and reference it using [NPM](https://docs.npmjs.com/cli/npm).
+When you want to call a Javascript library from your Client, it is easy to import and reference it using [NPM](https://docs.npmjs.com/cli/npm).
```

Comment:
Can you make it a capital S in JavaScript? On this line and the one below. Cheers.